cpp/src/groupby/hash/groupby.cu

```diff
@@ -84,9 +94,114 @@ bool constexpr is_hash_aggregation(aggregation::Kind t)
   // return array_contains(hash_aggregations, t);
   return (t == aggregation::SUM) or (t == aggregation::MIN) or (t == aggregation::MAX) or
          (t == aggregation::COUNT_VALID) or (t == aggregation::COUNT_ALL) or
-         (t == aggregation::ARGMIN) or (t == aggregation::ARGMAX);
+         (t == aggregation::ARGMIN) or (t == aggregation::ARGMAX) or (t == aggregation::MEAN) or
+         (t == aggregation::SUM_OF_SQUARES) or (t == aggregation::STD) or
+         (t == aggregation::VARIANCE);
 }
 
+class hash_compound_agg_finalizer final : public cudf::detail::aggregation_finalizer {
+  size_t i;
```

Comment:
`i` is not descriptive enough for a class data member.

Reply:
fixed.